src/replica/bulk_load/replica_bulk_loader.cpp

```diff
@@ -414,6 +414,9 @@ error_code replica_bulk_loader::download_sst_files(const std::string &remote_dir
                     remote_dir, local_dir, f_meta.name, fs, f_size);
                 const std::string &file_name =
                     utils::filesystem::path_combine(local_dir, f_meta.name);
+                if (ec == ERR_PATH_ALREADY_EXIST) {
```

Comment:
Why we don't need to verify the existing file here?

Reply:
Fixed